Makefile

```diff
@@ -0,0 +1,17 @@
+
+REGISTRY=gcr.io/ossf-malware-analysis
+
+.PHONY: all all_docker analysis_docker
+
+all: all_docker
+
+all_docker:
+	bash build/build_docker.sh
+
+analysis_docker:
+	docker build -t ${REGISTRY}/analysis -f cmd/analyze/Dockerfile .
```

Comment:
Any reason to delete the bash script and replace things inline here? May be better to be consistent with the other docker build scripts here.

Reply:
From what I understood talking to @calebbrown, we actually wanted to move towards everything being in the Makefile. That way it's a central point for dev tasks. I just left the build_docker.sh script there for now, so we can at least start with something in the Makefile. But if this is too incremental of a change currently, I can move the `build_docker` script into here.

Reply:
Got it, this SGTM then.